tests/input_files/f3_cha_xwall_ifpack.xml

```diff
@@ -0,0 +1,7 @@
+<ParameterList name="Ifpack">
+  <!-- General settings for Ifpack preconditioners -->
+  <Parameter name="Preconditioner type" type="string" value="ILU"/>
+  <Parameter name="Overlap" type="int" value="5"/>
```

Comment:
If this is required to get the test passing, then let's do it. Just FYI: this is a huge overlap, so I'd not recommend that in day-to-day practice.